js/packages/web/src/views/auctionCreate/index.tsx

```diff
@@ -312,6 +332,40 @@ export const AuctionCreateView = () => {
         ? attributes.items[0]
         : attributes.participationNFT,
       QUOTE_MINT,
+      new PriceFloor({
+        type: attributes.priceFloor
+          ? PriceFloorType.Minimum
+          : PriceFloorType.None,
+        hash: new U832ArrayBorshHack({
```

Comment:
Should we create a static method on `U832ArrayBorshHack` so we could do something like `U832ArrayBorshHack.fromArray(asArray)` or `U832ArrayBorshHack.create(asArray)`?

Reply:
yeah not a bad call. @bartosz-lipinski is gonna first see if there is a way borsh can just handle this case better so we don't even need this, but if there isn't, I'll add that.